root/static/styles/widgets.less

```diff
@@ -303,6 +303,11 @@ div.autocomplete2 {
         display: flex;
         flex-direction: row;
 
+        &.required::before {
+            position: absolute;
+            .asterisk();
+        }
+
```

Comment:
We're probably going to want this for *every* required field, not just autocomplete ones, so I would move this to a more generic place (`forms.less`?)

We don't have to implement it everywhere *yet* I guess, but it wouldn't hurt either if trivial. I'd possibly just put it next to 

```
label.required,
div.required {
    font-weight: bold;
}
```
in `forms.less` already, if that doesn't break anything.